Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1771,9 +1772,12 @@ extension BrowserViewController: ContextMenuHelperDelegate {
             dialogTitle = url.absoluteString
             let newTabTitle = NSLocalizedString("Open In New Tab", comment: "Context menu item for opening a link in a new tab")
             let openNewTabAction =  UIAlertAction(title: newTabTitle, style: UIAlertActionStyle.Default) { (action: UIAlertAction!) in
-                let request =  NSURLRequest(URL: url)
-                let tab = self.tabManager.addTab(request: request)
+                self.showToolbars(animated: self.headerConstraintOffset != 0, completion: { _ in
+                    let request =  NSURLRequest(URL: url)
+                    let tab = self.tabManager.addTab(request: request)
```

Comment:
While here, might as well remove `let tab =` since it's not used. Also, looks like we have an extra space before `NSURLRequest`.